tests/data/report-specs/report_spec.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "test_report",
+    "version": "0.1",
+    "title": "Example report spec",
+    "description": "This is a small test report which is used to ensure report_spec.py is working",
+    "attributes": [
+        {
+            "description": "An attribute of type int",
+            "type": "int",
+            "id": "attribute1",
+            "name": "Attribute 1",
+            "format": "{:,d}"
+        },
+        {
+            "description": "An attribute of type float",
+            "type": "float",
+            "id": "attribute2",
+            "name": "Attribute 2",
+            "format": "{p:5g} %"
+        },
+        {
+            "description": "An attribute of type bool",
+            "type": "boolean",
+            "id": "attribute3",
+            "name": "Attribute 3",
+            "format": null
+        },
+        {
+            "description": "An attribute of type string",
+            "type": "string",
+            "id": "attribute4",
+            "name": "Attribute 4",
+            "format": null
+        }
+    ],
+    "tables": [
+        {
+            "id": "table1",
+            "title": "Table 1",
+            "description": "The first table",
+            "columns": [
+                {
+                    "header": "Column 1",
+                    "type": "int",
+                    "id": "column1",
+                    "description": "A column of type int",
+                    "format": "%d"
```

Comment:
Yeah, I missed that - I really need to have a function that validates the formats too.